client/lib/query-manager/test/index.js

```diff
@@ -16,6 +16,11 @@ describe( 'QueryManager', () => {
 
 	beforeEach( () => {
 		manager = new QueryManager();
+		sandbox.create();
```

Comment:
Hmm, do we need to explicitly call `sandbox.create` though? It's not clear why this is needed in addition to the `restore`. I can't seem to find any docs about it either.